api/scripts/getOldPolkadotRewardEras.js

```diff
@@ -115,17 +133,19 @@ function parseRewards(
             staked = exposure.own
           } else {
             const stakerExposure = exposure.others.find(
-              ({ who }) => who === delegator
+              ({ who }) => who.toString() === delegator
             )
 
             staked = stakerExposure ? stakerExposure.value : ZERO
+            staked = isHex(staked) ? new BN(staked.substr(2, staked.length - 2), 16) : new BN(staked)
           }
-
+          
           value = available
             .minus(validatorCut)
             .times(staked)
             .dividedBy(exposureTotal)
             .plus(validatorId === delegator ? validatorCut : ZERO)
+
```

Comment:
Why did you add these lines? I think it was better before

Reply:
True, removed!